demo/src/com/google/maps/android/utils/demo/GeoJsonDemoActivity.java

```diff
@@ -45,9 +46,9 @@ protected int getLayoutId() {
     @Override
     protected void startDemo() {
```

Comment:
The demo currently loads with the default `(0,0)` as the map center (left screenshot below).  This gives the appearance of an empty map.  I'd suggest recentering the map so you can see some of the markers on startup (for example, right screenshot):

![image](https://cloud.githubusercontent.com/assets/928045/22296417/17a31590-e2e8-11e6-854c-c54cb92e9d7f.png)